### ISSUE TITLE
Provide offline support option

### ISSUE THREAD

**Issue by LukeMauldin** [pizzamig/pot#112]
**Is your feature request related to a problem? Please describe.**
I am attempting to use pot in an environment that is not directly connected to the Internet so I get failures on these sorts of operations: https://github.com/pizzamig/pot/blob/b63e3c86045bd7dde7b47441c25359cf166f74a5/share/pot/common.sh#L837

**Describe the feature you'd like to have**
I would like for pot to offer a setting where I could override the default http location for FreeBSD images so I can point pot to a local copy that i have posted on my local http web server.

**Comment by pizzamig** [comment]
Hi. Thanks for reporting.
You would like to have a way to specify a different URL to download the base.txz files.

Do you also need the disable the checksum check?

**Comment by LukeMauldin** [comment]
Yes you are correct. It would also be nice to have a way to disable the checksums as well.
